playbooks/README.md

```diff
@@ -169,6 +169,52 @@ $ ansible-playbook -i inventory -e "selinux=permissive" playbooks/selinux.yml
 
 Be sure to update the [inventory file](../inventory) according to your OpenShift cluster configuration or use the file you used to deploy the cluster.
 
+### SR-IOV
+
+By default, ```kubevirt-ansible``` will deploy SR-IOV components (CNI, device
+plugin) when ```network-multus``` is used. (You can disable SR-IOV deployment
+with ```deploy_sriov_plugin``` variable.)
+
+Note: ```kubevirt-ansible``` will *not* configure your hosts for SR-IOV. You
+should enable IOMMU, SR-IOV in your BIOS and in kernel, plus enable the desired
+number of VFs on your PFs before starting ```kubevirt-ansible``` deployment.
+You should also load ```vfio-pci``` kernel module since Kubevirt uses VFIO to
+pass SR-IOV VFs through into virtual machines.
+
+SR-IOV device plugin uses ```/etc/pcidp/config.json``` file to determine which
+PFs should be used to allocate VFs from. By default, ```kubevirt-ansible```
+doesn't configure any PFs, meaning that the plugin has zero devices to manage.
+
+There are two ways to configure allocatable PFs:
+
+a) use ```sriov_pci_ids``` playbook variable to specify PFs, for example:
+
+$ ansible-playbook -i inventory -e@vars/all.yml -e "sriov_pci_ids=\"0000:05:00.0\",\"0000:05:00.1\"" playbooks/kubevirt.yaml
+
+Note: this won't work if you have different PCI IDs on different nodes.
+
+b) deploy Kubevirt as usual, then manually edit ```/etc/pcidp/config.json``` on
```

Comment:
with the configmap approach, changes to this file will be overwritten during sriov dp restart with configmap content. we can either change the initContainer to keep existing config.json or edit this documentation

Reply:
Indeed I should now document ConfigMap.